x-pack/plugins/index_management/public/application/components/component_templates/component_template_wizard/component_template_clone/component_template_clone.tsx

```diff
@@ -30,7 +30,7 @@ export const ComponentTemplateClone: FunctionComponent<RouteComponentProps<Param
 
   useEffect(() => {
     if (error && !isLoading) {
-      toasts.addError(error, {
+      toasts.addError({ ...error, name: '' } as Error, {
```

Comment:
Can we add a comment explaining why we need to override a possible `name` present on the `error`?

Reply:
Yes, good catch!